src/library/scala/collection/mutable/OpenHashMap.scala

```diff
@@ -64,7 +70,14 @@ extends AbstractMap[Key, Value]
   private[this] val actualInitialSize = OpenHashMap.nextPositivePowerOfTwo(initialSize)
 
   private var mask = actualInitialSize - 1
-  private var table : Array[Entry] = new Array[Entry](actualInitialSize)
+
+  /** The hash table.
+    * 
+    * The table's entries are initialized to `null`, indication of an empty slot.
+    * A slot is either deleted or occupied if and only if the entry is non-`null`.
```

Comment:
> "A slot is either deleted or occupied."

That's not true. A slot can also be empty.

> "It is occupied if and only if [...] entry's keys is non-`null`."

I try to define the distinction between the deleted and occupied states close to the data that distinguishes them, i.e., in the `OpenEntry` docs. Also, it's not clear to me whether this class intends to support `null` keys.